staging/src/k8s.io/api/testdata/HEAD/node.k8s.io.v1alpha1.RuntimeClass.json

```diff
@@ -0,0 +1,60 @@
+{
+  "kind": "RuntimeClass",
```

Comment:
results of `UPDATE_COMPATIBILITY_FIXTURE_DATA=true go test ./vendor/k8s.io/api -run //HEAD` to generate fixtures

Reply:
actually, these should have already been generated... fixed separately in master for picking to 1.15 in https://github.com/kubernetes/kubernetes/pull/79029

Reply:
@liggitt - thanks, reverting this and rebasing it on your PR after it merges